hapi/src/utils/lifebankcode.js

```diff
@@ -76,7 +76,7 @@ const upLifebank = (
         data: {
           account,
           lifebank_name: name,
-          has_immunity_test: has_immunity_test || false,
+          hasImmunityTest: hasImmunityTest || false,
```

Comment:
same here, the smart contract expect `has_immunity_test` not `hasImmunityTest` so you should use `has_immunity_test: hasImmunityTest || false,`